contracts/Shortfall/Shortfall.sol

```diff
@@ -312,17 +349,17 @@ contract Shortfall is Ownable2StepUpgradeable, AccessControlledV8, ReentrancyGua
     }
 
     /**
-     * @notice Update next bidder block limit which is used determine when an auction can be closed
-     * @param _nextBidderBlockLimit  New next bidder block limit
+     * @notice Update next bidder block or timestamp limit which is used determine when an auction can be closed
+     * @param _nextBidderBlockOrTimestampLimit  New next bidder block limit
      * @custom:event Emits NextBidderBlockLimitUpdated on success
      * @custom:access Restricted by ACM
      */
-    function updateNextBidderBlockLimit(uint256 _nextBidderBlockLimit) external {
+    function updateNextBidderBlockLimit(uint256 _nextBidderBlockOrTimestampLimit) external {
         _checkAccessAllowed("updateNextBidderBlockLimit(uint256)");
-        require(_nextBidderBlockLimit != 0, "_nextBidderBlockLimit must not be 0");
+        require(_nextBidderBlockOrTimestampLimit != 0, "_nextBidderBlockOrTimestampLimit must not be 0");
         uint256 oldNextBidderBlockLimit = nextBidderBlockLimit;
```

Comment:
We could rename also `oldNextBidderBlockLimit`, or we could even avoid it emitting the event before the assignation

Reply:
Done. I preferred emitting the event before the assignation. https://github.com/VenusProtocol/isolated-pools/pull/324/commits/5845b1e659d4e7228d4c1aeccb2d0172d0a09add